husky_directory/blueprints/saml.py

```diff
@@ -34,10 +31,12 @@ def process_saml_request(self, request: Request, session: LocalProxy, **kwargs):
         self.logger.info(f"Signed in user {session['uwnetid']}")
         relay_state = request.form.get("RelayState")
 
-        if relay_state and relay_state.startswith("/"):
-            return redirect(urljoin(request.url_root, relay_state))
-
-        return redirect("/")
+        if relay_state:
```

Comment:
Not sure I'm putting this in the right line but...
I tested the sign in link and not sure I'm getting the right behavior.
1. Nav to https://uw-directory.iamdev.s.uw.edu/
2. click the _Sign in for student search_ link
3. complete the sign in
4. I am back on the search page, no sign in link.
5. I do my search (name, goulter)
6. Results page has the _Sign in for student search_ link

Reply:
hmm looks like the session isn't persisting. this is probably because it's using a hacky local cache, and the new request doesn't have access to it.

Reply:
that's a bigger issue that is probably going to result in a shared redis cache, so i would consider out of scope for right now.

Reply:
sounds good.